firefox/background.js

```diff
@@ -552,19 +552,18 @@ var url = "";
 //create empty artifact variable
 var artifact = "";
 
-function sanitizeArtifact(artifact) { 
-    while(String(artifact).includes("[.]")) {
-            artifact = String(artifact).replace("[", "").replace("]", "");
+function sanitizeArtifact(artifact) {
+    while(artifact.includes("[.]")) {
+        artifact = artifact.replace("[.]", ".");
     }
 
```

Comment:
Let's change these replace statements below to `hxxp://` and `hxxps://`

Reply:
I haven't thought about cases where "hxxp" or "hxxps" can be in domain or path. Of course I'm gonna add that.